### DISCUSSION TITLE
Plugin auto inputing data!

### DISCUSSION THREAD

**Discussion by dxcore35** [mdelobelle/metadatamenu#353] [Tips and Tricks]
I haven't defined anywhere it should give me current date, nor I defined that it will auto fill it for me. What is going on?

https://user-images.githubusercontent.com/7279156/220711248-705f86c9-3cce-43a5-99bb-be0641e8bb10.mov

https://user-images.githubusercontent.com/7279156/220711260-8385c7d6-0e0c-4a6d-8310-ee454de20e60.mov

**Comment by dxcore35** [comment]
Wau this is it!

**Comment by queezythegreat** [comment]
There is a solution with Various Complements version [8.0.0-beta4](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/discussions/183).  The catch is that you need to install it using BRAT plugin, as the current version you can install is 7.3.0.

You can suppress autocomletion using a Regex:
<img width="790" alt="image" src="https://user-images.githubusercontent.com/552992/221559418-d9306b4a-13f5-4149-b53a-80b8f670d1fb.png">

The regex I used, which matches when you are in a Inline Field:
```
[^([:]+::[^\]\)\n]+$
```

See thread: https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/issues/162#issuecomment-1446202871
